controllers/funding/index.js

```diff
@@ -30,4 +31,23 @@ router.get('/', sMaxAge('30m'), injectHeroImage('active-plus-communities'), inje
     });
 });
 
+router.get(
+    '/past-grants',
+    injectCopy('funding.pastGrants'),
+    injectHeroImage('active-plus-communities'),
+    async (req, res, next) => {
+        let caseStudies = await contentApi.getCaseStudies({
+            locale: req.i18n.getLocale()
+        });
+
+        // Shuffle the valid case studies and grab the first few
+        caseStudies = take(shuffle(caseStudies.filter(c => c.grantId)), 3);
```

Comment:
I think lodash has a dedicated method called `sampleSize` which does this for you.